ts/Stock/Indicators/DEMA/DEMAIndicator.ts

```diff
@@ -0,0 +1,256 @@
+/* *
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+import type {
+    DEMAOptions,
+    DEMAParamsOptions
+} from './DEMAOptions';
+import type DEMAPoint from './DEMAPoint';
+import type IndicatorValuesObject from '../IndicatorValuesObject';
+import type LineSeries from '../../../Series/Line/LineSeries';
+import BaseSeries from '../../../Core/Series/Series.js';
+const {
+    seriesTypes: {
+        ema: EMAIndicator
+    }
+} = BaseSeries;
+import RequiredIndicatorMixin from '../../../Mixins/IndicatorRequired.js';
+import U from '../../../Core/Utilities.js';
+const {
+    correctFloat,
+    isArray,
+    merge
+} = U;
+
+
+/**
+ * Internal types
+ * @private
+ */
+declare global {
+    namespace Highcharts {
+        interface DEMAIndicatorLinkedParentSeries extends LineSeries {
+            EMApercent?: number;
+        }
+    }
+}
```

Comment:
This should be deleted. Replace the DEMAIndicatorLinkedParentSeries type with `LineSeries` and add in this place:
```ts
/* *
 *
 *  Declarations
 *
 * */

declare module '../../../Core/Series/SeriesLike' {
    interface SeriesLike {
        EMApercent?: number;
    }
}
```